### PR TITLE
update default version of API to v1 instead of v1beta3 [#101]

### PR DESCRIPTION
**Pull request by jonmoter**
Updates defaults in code, as well as tests.

See issue #101.

### CODE REVIEW

**Comment by abonas** [comment]
@jonmoter Hi, and thanks a lot for your contribution.
Couple of comments from briefly looking at this:
all test json files names remained with "b3" name which stands for beta 3.
a. it makes sense to rename them as part of this change.
b. is the format kept 100% identical in k8s between beta3 to v1? so you can simply change the api version in the json and that would be enough? (no other fields added, fields renamed or moved between sections, etc.?)
c. in order to link properly the referenced github issue, please make sure to follow this convention in your commit comment:
https://help.github.com/articles/closing-issues-via-commit-messages/

**Comment by abonas** [comment]
cc @simon3z

**Comment by jonmoter** [comment]
Okay, I've renamed the files to remove the "_b3" in the filename.

I went over the Kubernetes docs describing [changes from v1beta3 to v1](https://github.com/kubernetes/kubernetes/blob/master/docs/api.md#v1-conversion-tips-from-v1beta3), and found a few small changes that were required. I updated the tests to reflect that.

Let me know if you have any other feedback.

**Comment by abonas** [comment]
> c. in order to link properly the referenced github issue, please make sure to follow this convention in your commit comment:
> https://help.github.com/articles/closing-issues-via-commit-messages/

@jonmoter LGTM except for not linking commit msg

**Comment by jonmoter** [comment]
Would you like me to rebase with the commit message fixed?

**Comment by abonas** [comment]
You don't need to rebase for that, just fix the commit msg, then repush.
You can also squash the 3 commits, they all handle the same topic

**Comment by jonmoter** [comment]
@abonas okay, i've squashed the commits and fixed the commit message.

File: test/test_node.rb [hunk 1]
```diff
@@ -4,10 +4,10 @@
 class TestNode < MiniTest::Test
   def test_get_from_json_v3
```
Comment (abonas) [inline test/test_node.rb]: missed a rename here in the method name

File: test/test_pod.rb [hunk 2]
```diff
@@ -4,10 +4,10 @@
 class TestPod < MiniTest::Test
   def test_get_from_json_v3
```
Comment (abonas) [inline test/test_pod.rb]: missed a rename here in the method name

File: test/test_replication_controller.rb [hunk 3]
```diff
@@ -4,10 +4,10 @@
 class TestReplicationController < MiniTest::Test
   def test_get_from_json_v3
```
Comment (abonas) [inline test/test_replication_controller.rb]: missed a rename here in the method name

**Comment by abonas** [comment]
@jonmoter great. I just noticed couple of method still refer to v3, please remove that too and it's good to be merged.

**Comment by jonmoter** [comment]
D'oh, good catch.  I'll fix and squash.

**Comment by jonmoter** [comment]
@abonas fixed.  Should be good to go.

**Comment by abonas** [comment]
:+1:

**Pull request merged**